index.html

```diff
@@ -86,8 +87,8 @@ <h2 class="head2">Tonic</h2>
                         <div class="link3">javaScript</div>
                     </li>
                 </ul>
-                <button class="see-project1 unique2">
-                    <div class="see-project">See Project</div>
+                <button class="see-project1 project-buttons unique2">
+                    <div class="see-project" id="sec2-project">See Project</div>
```

Comment:
[REQUIRED] None of your popups is working. No matter how much I click the `see project` buttons I get nothing.

Reply:
@fobadara Hello,

They are all actually working. I just tried them now; both in mobile and desktop view.

![image](https://user-images.githubusercontent.com/112436398/209404224-6f26b087-fad1-4fdb-a6bc-c048ebb72741.png)